package.json

```diff
@@ -153,21 +153,21 @@
     "moment": "2.30.1",
     "moment-timezone": "0.5.46",
     "omit-empty-es": "1.2.0",
-    "patch-package": "6.5.1",
+    "patch-package": "^8.0.0",
     "postcss": "8.4.49",
     "postcss-styled-syntax": "^0.7.0",
     "postcss-syntax": "^0.36.2",
     "postcss-value-parser": "4.2.0",
     "pptr-testing-library": "0.8.0",
     "prettier": "2.8.8",
+    "prop-types": "^15.8.1",
```

Comment:
Prop types are [not supported](https://react.dev/blog/2024/04/25/react-19-upgrade-guide#removed-proptypes-and-defaultprops) in React v19 so I guess we should not be adding this dependency.

Reply:
we have a number of tests that are js files with `propTypes` declared, and updating those to typescript seemed out of the scope of this (already large) PR.  

That is why `prop-types` is a dependency at the workspace root, as it only needs to be available as a dev dependency